packages/fractal-page-object/API.md

```diff
@@ -93,21 +94,18 @@ class Page extends PageObject {
 
 setRoot(rootElement);
 
-let page = new Page();
-page.list.elements; // rootElement.querySelectorAll('.list')
-```
-
-```javascript
-import { PageObject, selector, setRoot } from 'fractal-page-object';
-
-class Page extends PageObject {
-  list = selector('.list');
-}
-
-setRoot(rootElement);
-
-let page = new Page('.container');
-page.list.elements; // rootElement.querySelectorAll('.container .list')
+// rootElement.querySelectorAll('.list')
+new Page().list.elements;
+// rootElement.querySelectorAll('.container .list')
+new Page('.container').list.elements;
+// rootElement.querySelectorAll('.container')[0].querySelectorAll('.list')
+new Page('.container', null, 0).list.elements;
+// document.body.querySelectorAll('.list');
+new Page('', document.body).list.elements;
+// document.body.querySelectorAll('.container .list');
+new Page('.container', document.body).list.elements;
+// document.body.querySelectorAll('.container')[1].querySelectorAll('.list');
+new Page('.container', document.body, 1).list.elements;
```

Comment:
🎉  but same question -> is this `.elements` at the end of the commented example of the result?